### PR TITLE
Moving Domain map to the short use cases section

### PR DESCRIPTION
**Pull request by philarcher**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-use-cases/pull/142.html" title="Last updated on Feb 9, 2021, 5:54 PM UTC (6bb8ee6)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-use-cases/142/6020f7e...6bb8ee6.html" title="Last updated on Feb 9, 2021, 5:54 PM UTC (6bb8ee6)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -633,6 +633,16 @@ <h2 id="concepts">Concepts of Decentralized Identity</h2>
 
       <section id="uc">
         <h2>Use Cases</h2>
+	      <p>We present use cases in two formats: short use cases are in this section and extended use cases, 
+		      that provide more detail, are presented in <a href="#focalUseCases"></a>. The diagram below provides
```
Comment (TallTed) [inline index.html:637]:
```suggestion
	      <p>We present use cases in two formats: short use cases in this section; and extended use cases, 
		      providing more detail, in <a href="#focalUseCases"></a>. The diagram below provides 
```

File: index.html [hunk 2]
```diff
@@ -633,6 +633,16 @@ <h2 id="concepts">Concepts of Decentralized Identity</h2>
 
       <section id="uc">
         <h2>Use Cases</h2>
+	      <p>We present use cases in two formats: short use cases are in this section and extended use cases, 
+		      that provide more detail, are presented in <a href="#focalUseCases"></a>. The diagram below provides 
+		      a visual guide to the different domains where Decentralized Identity is seen as important to people
```
Comment (TallTed) [inline index.html:638]:
```suggestion
		      a visual guide to different domains where Decentralized Identity is seen as important to people,
```

**Comment by philarcher** [comment]
Thanks @TallTed - I appreciate the help here. I have made those edits directly in the document. I can't do it here as the PR is merged and closed.

**Comment by TallTed** [comment]
@philarcher -- Thanks for noticing my after-merge suggestions. I wish Github made it more blatant that the merge had already happened -- even to disallowing suggestions entirely, with some sort of error feedback to the suggestion submitter -- as I too frequently fail to notice the purple pill at the top of the page!

**Pull request merged**